.github/issue-labeler.yml

```diff
@@ -0,0 +1,146 @@
+assessment:
+  - assessment
+  - bonus
+  - complaint
+  - submission
+  - feedback
+  - grading
+  - grade
+  - score
+  -
+  - caseSensitive: false
+
+athena:
+  - athena
+  - caseSensitive: false
+
+atlas:
+  - atlas
+  - competency
+  - competencies
+  - relation
+  - source
+  - knowledge area
+  - competency catalog
+  - learning path
+  - learner profile
+  - competency catalog
+  - science event
+  - caseSensitive: false
+
+buildagent:
+  - buildagent
+  - build agent
+  - buildjob
+  - build job
+  - build result
+  - caseSensitive: false
+
+communication:
+  - communication
+  - conversation
+  - notification
+  - agreement
+  - faq
+  - post
+  - reaction
+  - caseSensitive: false
+
+core:
+  - user-management
+  - course
+  - authority
+  - data export
+  - migration
+  - user
+  - group
+  - caseSensitive: false
+
+exam:
+  - exam
+  - exercisegroup
+  - student exam
+  - suspicious
```

Comment:
I think this is about "suspicious behavior". I'd use the whole term to make it less like to cause false positives